include/PR/gbi.h

```diff
@@ -1361,7 +1361,7 @@ typedef struct {
 
 typedef union {
     Light_t	l;
-    //long long int	force_structure_alignment[2];
+    long long int	force_structure_alignment[2];
```

Comment:
Was this commented out in the original source or

Reply:
no, we had commented it out for reasons based on an assumption that proved to be incorrect. so we're able to make it go back to how it was, this way